pages/blog/release-notes-3.0.0.md

```diff
@@ -0,0 +1,247 @@
+### Cleaning up the root object
+There was two meta information lingering in the root of the AsyncAPI object, which did not make much sense since we have the `info` object for all the meta information.
+
+Therefore the root `tags` and `externalDocs` have been moved to the info object.
+
+```
+asyncapi: 3.0.0
+info:
+  ...
+  externalDocs:
+    description: Find more info here
+    url: https://www.asyncapi.org
+  tags:
+    - name: e-commerce
+...
+```
+
+Pull request: https://github.com/asyncapi/spec/pull/794
+### Splitting out server URL into host and pathname
+There have been some confusion about what `url` of a server should contain, is it both protocol + host + path? What about the protocol field then? Therefore the host, path, and protocol are all individual fields you defined.
+
+```
+asyncapi: 3.0.0
+servers:
+  localhost:
+    host: localhost
+    path: /api/v1,
+    protocol: mqtt
+```
+
+Issues: [#547](https://github.com/asyncapi/spec/issues/547), [#274](https://github.com/asyncapi/spec/issues/274) | Pull request: https://github.com/asyncapi/spec/pull/888
+### More reusable objects in components
+This is a bit of a mixture between some of the features, that all added a little to this. It's now possible to add more stuff under components:
+- External docs
+- Tags
+- replyAddresses
+- replies
+- operations
+- channels
+
+```
+asyncapi: 3.0.0
+components:
+  ...
+  replies:
+    ...
+  replyAddresses:
+    ...
+  tags: 
+    ...
+  externalDocs:
+    ...
+  operations:
+    ...
+  channels:
+    ...
+```
+
+Issues: [#829](https://github.com/asyncapi/spec/issues/829) | Pull request: https://github.com/asyncapi/spec/pull/847, https://github.com/asyncapi/spec/pull/792, https://github.com/asyncapi/spec/pull/806,  https://github.com/asyncapi/spec/pull/827 | Migration details: 
+
+### New trait behavior
+
+TODO
+
+### Schema format and payload definition
+With schemas, one thing that has always been impossible was reusing schemas with different schema formats. Thats because the schema format information is part of the message object, and the schema object, well schema object. That means that if you reference a Schema object, it has no information about the schema format.
+
+
+TODO
+
+Issues: [#829](https://github.com/asyncapi/spec/issues/829) | Pull request: https://github.com/asyncapi/spec/pull/847
+
+## Tooling support
+The following official AsyncAPI tools are already updated to support 3.0.0 version of the specification:
+
+TODO
+
+Last but not least is the AsyncAPI Studio. Check out the Studio with this example.
+
+## Acknowledgements
+Spec 3.0 have been a massive undertaking, so I would like to say a huge thank you! to everyone who have been involved, maybe you commented on your views, added to discussions, joined the live meetings, championed changes, reviewed proposed changes, list goes on, this section is for you!
+
+Thank you, xxxxxxxxxxxxxxxxx
```

Comment:
I want to spend an hour or two going through all the related issues and make sure we tag everyone who has contributed in any way or form to this release. 

Any objections?

Reply:
DO IT BRO, THE WORLD AWAITS